manifests/jenkins-with-cert.yaml

```diff
@@ -63,7 +63,7 @@ objects:
         dockerStrategy:
           from:
             kind: ImageStreamTag
-            name: jenkins-agent-base:latest
+            name: jenkins-agent-base:scheduled-upgrade
```

Comment:
I don't see `scheduled-upgrade` mentioned in the [docs](https://docs.redhat.com/en/documentation/openshift_container_platform/4.13/html/jenkins/important-changes-to-openshift-jenkins-images#customizing-the-jenkins-image-stream-tag_important-changes-to-openshift-jenkins-images) you linked but I do see `scheduled-upgrade-redeploy`. did you meant `scheduled-upgrade`?

Reply:
I did mean `scheduled-upgrade`. I'm a bit confused by it, but the `jenkins` imagestream uses [scheduled-upgrade-redeploy](https://github.com/openshift/jenkins/blob/401beedacfbffb3ff8b872c41deb87ce66c21b48/openshift/imagestreams/jenkins-rhel.yaml#L73) while the `jenkins-agent-base` imagestream uses the [scheduled-upgrade](https://github.com/openshift/jenkins/blob/83137040b1d89c1d0370c48ee7574e34d0014be4/openshift/imagestreams/jenkins-agent-base-rhel.yaml#L33) tag.

The wording in the PR might be a bit misleading. I chose to use `scheduled-upgrade` in the title to shorten it.